### ISSUE TITLE
after update to 3.1 symlink crashed

### ISSUE THREAD

**Issue by bartek1234321** [phiamo/MopaBootstrapBundle#1188]
When I type php app/console mopa:bootstrap:symlink:less below error occures:

RecursiveDirectoryIterator::__construct(C:\wamp\www\xxxx\vendor/mopa/bootstrapbundle/Mopa/Bundle/BootstrapBundle\Resources/public/bootstrap..\C:\wamp\www\xxxx\vendor/twbs/bootstrap,C:\wamp\www\xxxx\vendor/mopa/bootstrapbundle/Mopa/Bundle/BootstrapBundle\Resources/public/bootstrap..\C:\wamp\www\xxxx\vendor/twbs/bootstrap): Nazwa pliku, nazwa katalogu lub składnia etykiety woluminu jest niepoprawna. (code: 123)

Can anyone help?

**Comment by jonny827** [comment]
@bartek1234321 You need to use bin/console as there is no console executable in the app directory.    Try that and let me know if you still have any issues.  Also, where did you get app/console,  is it in the projects documentation somewhere? If so, please link me to the page so I can get it fixed please.

**Comment by bartek1234321** [comment]
I use symfony 2.8 and i don't have console i bin directory, some hours ago i was using s 2.6 and everything worked correctly ;)

**Comment by bartek1234321** [comment]
temporary solution ;) cp -Rv vendor/twbs/bootstrap vendor/mopa/bootstrap-bundle/Mopa/Bundle/BootstrapBundle/Resources/public/

**Comment by phiamo** [comment]
Hi bartek,
My wife just translated the error message for me and it looks for me like
directory seperators are mixed with slashes and backslashes...
Could you confirm?
Maybe some part or the command is not using Windows compatible
DIRECORY_SEPERATOR constant?

Am 24.08.2016 22:30 schrieb "bartek1234321" <notifications@github.com>:

temporary solution ;) cp -Rv vendor/twbs/bootstrap
vendor/mopa/bootstrap-bundle/Mopa/Bundle/BootstrapBundle/Resources/public/

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/phiamo/MopaBootstrapBundle/issues/1188#issuecomment-242197921>,
or mute the thread

**Comment by jonny827** [comment]
@bartek1234321    I would recommend upgrading to Symfony version 3.1.x if you can. (2.8 and 3.0 are basically the same minus removing of backwards compatibility going forward)   This eliminates a possible cause of error.

Secondly, do you have LESS properly configured as per here [MopaBootrapBundle LESS Install Instructions](https://github.com/phiamo/MopaBootstrapBundle/blob/master/Resources/doc/install/2-less-installation.md)

3) I found two links for you to read - this one discusses YUI Compressor Issues (which would make since)  since you just recently upgraded Symfony to 2.8 from 2.6,  Maybe try a new version of YUI Compressor -- [YUI Compressor Errors](https://github.com/kriswallsmith/assetic/issues/536)

4) Here is the second link -- may not be useful anymore but it is possible a configuration is off somewhere -- a bit dated but similar issue [Dumping Assets - Runtime Error - MopaBootstrapBundle](http://stackoverflow.com/questions/10451697/when-dumping-assetic-using-mopabootstrapbundle-runtime-exception-occurs-the-f)

So in short, upgrade to the latest of Symfony if you can, and upgrade YUI as well and ensure your NodeJS and Symfony configuration is correct.

**Comment by bartek1234321** [comment]
@phiamo yes I confirm it she's right. it's look like but which part? ;) I use only php app/console mopa:bootstrap:symlink:less exactly symfony mopa:bootstrap:symlink:less because I use intellij + symfony plugin + command line tool plugin. But none of them works correctly. Before update symfony it worked ;)

@jonny827 I will upgrade symfony to 3.1 but I'm doing it step by step so: 2.6 -> 2.8 -> 3.x
in 3.x we have other directory structure ;) less is correctly configured because everything else works fine. i do not use yui compressor i use uglifycss and uglifyjs2 is it wrong?

thanks for you help and time! i will upgrade to 3.1 soon and then check is it work.
